delta/utils/solver/estimator_solver.py

```diff
@@ -86,8 +86,12 @@ def _model_fn(features, labels, mode, params):
 
       if mode == utils.INFER:
         softmax = tf.nn.softmax(logits, name='softmax_output')
+        if self.config['solver']['postproc'].get('output_softmax', False):
```

Comment:
will broken other code.

Reply:
Okay. Tested on voxceleb, it works fine.
I'll revert it however.

Reply:
Commit reverted.